redash/handlers/users.py

```diff
@@ -31,17 +32,17 @@
 
 # Ordering map for relationships
 order_map = {
-    "name": "name",
-    "-name": "-name",
-    "active_at": "active_at",
-    "-active_at": "-active_at",
-    "created_at": "created_at",
-    "-created_at": "-created_at",
-    "groups": "group_ids",
-    "-groups": "-group_ids",
+    "name": [asc(models.User.name)],
```

Comment:
When defining `order_map`, `asc()` may or may not be present... Please standardize the writing style.

Reply:
@masayuki038 what do you mean?

Reply:
Sorry, I didn't explain well. 
In the Dashboard case, it looks like that `asc()` is omitted. However, the above code, `asc()` is not omitted. So, I think that it is good to standardize the writing style.

```
order_map = {
    "name": [
        func.lower(models.Dashboard.name),
        models.Dashboard.created_at,
        models.Dashboard.slug,
    ],
```

Reply:
removed `asc`

Reply:
Thanks. Confirmed.